src/Crdt/CrdtConfig.cs

```diff
@@ -48,10 +61,22 @@ private void JsonTypeModifier(JsonTypeInfo typeInfo)
 
 public class ChangeTypeListBuilder
 {
+    private bool _frozen;
+
+    public void Freeze()
+    {
+        _frozen = true;
+    }
+
+    private void CheckFrozen()
+    {
+        if (_frozen) throw new InvalidOperationException("ObjectTypeListBuilder is frozen");
```

Comment:
Wrong type name. Could use the type name directly to avoid missed refactoring later.